flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,41 @@
+flink-connector-hive
+Copyright 2014-2019 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.hadoop:hadoop-common:2.7.2
```

Comment:
how problematic would it be to not bundle hadoop and rely on our usual approach for hadoop (i.e. let the user provide it)?

Reply:
Hive metastore 2.x depends on `org.apache.hadoop.crypto` package in hadoop-common (hive-metastore 2.3.4 -> hadoop-common 2.7.2, hive-metastore 2.1.1 -> hadoop-common 2.6.1), and that package is only available since hadoop 2.6.

My main concern is that, according to https://flink.apache.org/downloads.html, we still support hadoop 2.4. If users provide that, then hive-metastore won't be able to run. I actually wonder if Flink 1.9.0 is a good time to cut off support for hadoop 2.4? If so, I think it'll be a big help and we don't need to bundle hadoop-common anymore.

What do you think?

Reply:
I think it would be fair to say that users must provide hadoop 2.6 to use this feature.

If a user were to use hadoop 2.4, while this connector bundles 2.7, you can easily end up with ill-defined behavior anyway; I don't think we really have to support this case.
(In case you're wondering, this is not an issue for our filesystems since they relocate hadoop completely)